plugin/build_baron.go

```diff
@@ -149,94 +40,13 @@ func (bbp *BuildBaronPlugin) GetPanelConfig() (*PanelConfig, error) {
 	}, nil
 }
 
-// IsWebhookConfigured webhook will can be retrieved from project or admin config depending on PluginAdminPageDisabled flag
-// if deriving from project config, we first try to retrieve webhook config prom project parser config, otherwise we fallback to project page settings
-// version is needed to retrieve last good project config, if version is not available/empty when calling this function we must first retrieve it
-func IsWebhookConfigured(project string, version string) (evergreen.WebHook, bool, error) {
-	var webHook evergreen.WebHook
-	flags, err := evergreen.GetServiceFlags()
-	if err != nil {
-		return evergreen.WebHook{}, false, err
-	}
-	if flags.PluginAdminPageDisabled {
-		projectRef, err := model.FindMergedProjectRef(project, version, true)
-		if err != nil || projectRef == nil {
-			return evergreen.WebHook{}, false, errors.Errorf("Unable to find merged project ref for project %s", project)
-		}
-		webHook = projectRef.TaskAnnotationSettings.FileTicketWebhook
-	} else {
-		bbProject, _ := BbGetProject(evergreen.GetEnvironment().Settings(), project, "")
-		webHook = bbProject.TaskAnnotationSettings.FileTicketWebhook
-		if webHook.Endpoint != "" && bbProject.TicketCreateProject != "" {
-			return evergreen.WebHook{}, false, errors.Errorf("The custom file ticket webhook and the build baron TicketCreateProject should not both be configured")
-		}
-	}
-	if webHook.Endpoint != "" {
-		return webHook, true, nil
-	} else {
-		return evergreen.WebHook{}, false, nil
-	}
-}
-
-func BbGetConfig(settings *evergreen.Settings) map[string]evergreen.BuildBaronSettings {
-	bbconf, ok := settings.Plugins["buildbaron"]
-	if !ok {
-		return nil
-	}
-
-	projectConfig, ok := bbconf["projects"]
-	if !ok {
-		grip.Error("no build baron projects configured")
-		return nil
-	}
-
-	projects := map[string]evergreen.BuildBaronSettings{}
-	err := mapstructure.Decode(projectConfig, &projects)
-	if err != nil {
-		grip.Critical(errors.Wrap(err, "unable to parse bb project config"))
-	}
-
-	return projects
-}
-
-// BbGetProject retrieves build baron settings from project or admin config depending on PluginAdminPageDisabled flag.
-// Project parser config takes precedence, otherwise fallback to project page settings (found using version ID if given).
+// BbGetProject retrieves build baron settings from project settings.
+// Project page settings takes precedence, otherwise fallback to project config yaml.
 // Returns build baron settings and ok if found.
-func BbGetProject(settings *evergreen.Settings, projectId string, version string) (evergreen.BuildBaronSettings, bool) {
-	flags, err := evergreen.GetServiceFlags()
-	if err != nil {
+func BbGetProject(projectId string, version string) (evergreen.BuildBaronSettings, bool) {
```

Comment:
This function also should be in model now (and maybe renamed to make more sense, like GetBuildBaronSettings)